src/store/actions/movieDetailsAction.js

```diff
@@ -0,0 +1,16 @@
+import axios from "../axiosHelper";
+
+export const getMovieDetails = (id) => async (dispatch) => {
+  dispatch({
+    type: "LOADING_MOVIE_DETAILS",
```

Comment:
Consider creating a "actionTypes.js" under "src/store" where you would store all such string values in variables or under a dedicated object.
E.g. in "actionTypes.js" there would be an `const ACTION_TYPES = { ..., LOADING_MOVIE_DETAILS: "LOADING_MOVIE_DETAILS", ...};`
The benefit is that it allows you to avoid typos in an action name across the app.

Reply:
resolved